build.yaml

```diff
@@ -2625,7 +2625,7 @@ steps:
       cpu: '1'
     script: |
       export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
-      hailctl config set batch/bucket {{ global.hail_test_gcs_bucket }}
+      hailctl config set batch/remote_tmpdir gs://{{ global.hail_test_gcs_bucket }}/batch/
```

Comment:
Now that this uses `remote_tmpdir` this can instead use the recently-added `test_storage_uri`:

```
hailctl config set batch/remote_tmpdir {{ global.test_storage_uri }}/batch/
```